### PR TITLE
[CON-382] Migrate `StoreDetailsHeader` to Typescript

### PR DESCRIPTION
**Pull request by beau-charman**
Addresses https://aesoponline.atlassian.net/browse/CON-382

### CODE REVIEW

File: src/components/StoreDetailHeader/StoreDetailHeader.fixture.ts [hunk 1]
```diff
@@ -6,7 +6,7 @@ const StoreDetailHeaderFixture = {
     openingHours: 'Opening hours',
   },
   email: 'consectetur.elit@aesop.com',
-  location: 'Duis in massa sed, Suspendisse hendrerit',
+  location: 'Duis in massa sed, Suspendisse in massa sed hendrerit',
```
Comment (MkMan) [inline src/components/StoreDetailHeader/StoreDetailHeader.fixture.ts:9]: 🤔 😄

**Review by MkMan — APPROVED** [review]

File: src/components/StoreDetailHeader/StoreDetailHeader.fixture.ts (hunk 1 above)
Comment (beau-charman) [reply src/components/StoreDetailHeader/StoreDetailHeader.fixture.ts:9]: 😎

Comment (beau-charman) [reply src/components/StoreDetailHeader/StoreDetailHeader.fixture.ts:9]: Wanted to make the text longer to test multiple lines visually - more like an actual address

Comment (MkMan) [reply src/components/StoreDetailHeader/StoreDetailHeader.fixture.ts:9]: Ahhhh fair enough, that makes sense 😄

**Pull request merged**